datanode/gateway/graphql/resolvers.go

```diff
@@ -2193,9 +2216,65 @@ func (r *mySubscriptionResolver) Orders(ctx context.Context, filter *OrderByMark
 }
 
 func (r *mySubscriptionResolver) Trades(ctx context.Context, market *string, party *string) (<-chan []*types.Trade, error) {
+	markets := []string{}
+	parties := []string{}
+	if market != nil {
+		markets = append(markets, *market)
+	}
+
+	if party != nil {
+		parties = append(parties, *party)
+	}
+
 	req := &v2.ObserveTradesRequest{
-		MarketId: market,
-		PartyId:  party,
+		MarketId: markets,
+		PartyId:  parties,
```

Comment:
Shouldn't these fields be names MarketIds and PartyIds?

Reply:
Resolved in https://github.com/vegaprotocol/vega/pull/7979/commits/df717796c53dc24d1408a3826a473e8d96f1e07d